internal/pkg/event/event.go

```diff
@@ -52,17 +52,17 @@ type ObserveNotifier interface {
 
 // Embeddable is a type that implements sane defaults as an observer.
 type Embeddable struct {
-	channel Channel
-	types   []Type
+	Chan  Channel
+	types []Type
 }
 
 // Channel implements the Observer interface.
 func (e *Embeddable) Channel() Channel {
-	if cap(e.channel) == 0 {
-		e.channel = make(Channel, 20)
+	if cap(e.Chan) == 0 {
```

Comment:
should it be `e.Chan == nil`?

Reply:
🤔 not sure. I can check.

Reply:
`cap` seems to work: https://play.golang.org/p/IHJz1I74p0j

Reply:
what I meant is that unbuffered not-nil `channel` object will have `cap(e.Chan) == 0`, but this code will replace it with buffered channel. so `cap(nil) == 0` as well I guess, but this covers both. probably it's not important for this PR if we get a "bus" fix coming up.